megalinter/descriptors/xml.megalinter-descriptor.yml

```diff
@@ -8,15 +8,25 @@ file_extensions:
   - ".xml"
 linters:
   # XML LINT
-  - linter_name: xmllint
+  - class: XmlLintLinter
+    linter_name: xmllint
+    variables:
+      - name: XML_XMLLINT_AUTOFORMAT
+        default_value: "false"
+        description: If set to `true`, it will reformat and reindent the output
+      - name: XML_XMLLINT_INDENT
+        default_value: "  "
+        description: The number of indentation spaces when `XML_XMLLINT_AUTOFORMAT` is `true`
     linter_url: http://xmlsoft.org/xmllint.html
     linter_rules_url: http://xmlsoft.org/xmllint.html#diagnostics
     version_extract_regex: "(?<=libxml version )\\d+(\\d+)+"
     cli_lint_mode: list_of_files
+    cli_lint_fix_arg_name: "--format"
     cli_help_arg_name: "--help"
     help_command_return_code: 1
     examples:
       - "xmllint myfile.xml"
+      - "xmllint --format myfile.xml --output myfile.xml"
```

Comment:
this example is not existing, no ?
maybe better to write the real examples like `XML_XMLLINT_AUTOFORMAT xmllint myfile.xml myfile2` ?

Reply:
What do you mean it does not exist? I have based on the first example that was already there and looking at other descriptors, they also add similar examples.

I don't understand the example you give me either, that line is not a valid command.

Reply:
oops, i didn't understand , my bad ^^